frontend/package.json

```diff
@@ -51,6 +51,7 @@
 		"eslint-plugin-jest": "^25.2.2",
 		"eslint-plugin-vue": "^7.0.0",
 		"eslint-webpack-plugin": "^2.4.0",
+		"jest-mock-axios": "^4.6.1",
```

Comment:
Please reword your commit like: "Setup mock-axios in jest test"

Reply:
Ok